src/evaluation/Run.jl

```diff
@@ -115,8 +116,12 @@ function update_save_run!(session::ServerSession, notebook::Notebook, cells::Arr
 	else
 		# this code block will run cells that only contain text offline, i.e. on the server process, before doing anything else
 		# this makes the notebook load a lot faster - the front-end does not have to wait for each output, and perform costly reflows whenever one updates
-		offline_workspace = WorkspaceManager.make_workspace(notebook, false)
-		
+		# "A Workspace on the main process, used to prerender markdown before starting a notebook process for speedy UI."
+		offline_workspace = WorkspaceManager.make_workspace(
```

Comment:
this workspace is created in the server process, which is causing the following workspace's compiler options pwd become incorrect.